<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_by_sort</class>
 <widget class="QDialog" name="search_by_sort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>按种类查询总收支</string>
  </property>
  <widget class="QLabel" name="labelHint">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>？</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSort">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>62</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>种类</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_expense">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>shopping</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>savings</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>entertainment</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>food</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>others</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_income">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>salary</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bonus</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>rents</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>others</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_hint">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>总金额：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_num">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
